modules/installation-requirements-user-infra-ibm-z-kvm.adoc

```diff
@@ -91,36 +91,41 @@ On your {op-system-base} KVM host, set up:
 
 Each cluster virtual machine must meet the following minimum requirements:
 
-[cols="2,2,2,2,2",options="header"]
+[cols="2,2,2,2,2,2",options="header"]
 |===
 
 |Virtual Machine
 |Operating System
 |vCPU ^[1]^
 |Virtual RAM
 |Storage
+|IOPS
 
 |Bootstrap
 |{op-system}
 |4
 |16 GB
-|120 GB
+|100 GB
+|N/A
 
 |Control plane
 |{op-system}
 |4
 |16 GB
-|120 GB
+|100 GB
+|N/A
 
 |Compute
 |{op-system}
 |2
 |8 GB
-|120 GB
+|100 GB
+|N/A
 
 |===
+[.small]
 --
-^[1]^ 1 physical core (IFL) provides 2 logical cores (threads) when SMT-2 is enabled. The hypervisor can provide 2 or more vCPUs.
+1. 1 physical core (IFL) provides 2 logical cores (threads) when SMT-2 is enabled. The hypervisor can provide 2 or more vCPUs.
```

Comment:
```suggestion
1. One physical core (IFL) provides 2 logical cores (threads) when SMT-2 is enabled. The hypervisor can provide 2 or more vCPUs.
```

> In technical documentation, use words if a number occurs at the beginning of a sentence.

Consider using words instead of numbers for the rest of them. We were attempting to follow the earlier guidance to spell out numbers below 10.

> For numbers below 10, it is alternatively acceptable to use words instead of numerals. Within a publication, be consistent in your approach.